config.toml

```diff
@@ -18,6 +18,18 @@ allowed-github-orgs = [
     "rust-analyzer",
 ]
 
+allowed-domains = [
+    "https://(.*)rust-lang.org/?.*",
+    "https://areweasyncyet.rs/?.*",
+    "https://crates.io/?.*",
+    "https://docs.rs/?.*",
+    "https://github.com/rust-lang/rust/issues/35437",
```

Comment:
I'm curious: why also the URL of this github issue? :)

Reply:
Because someone set it as the URL on a repo. 😅 I don't want to allow all of GitHub with something like `https://github.com/?.*`, because that could be abused with typosquatting. But we can change this to allow anything under the `rust-lang` organization with a rule like `http://github.com/rust-lang/?.*`.